components/GlobalStyles.js

```diff
@@ -3,19 +3,34 @@ import { createGlobalStyle } from "styled-components";
 const GlobalStyles = createGlobalStyle`
     html,
     :root {
---color-background: #FFFFFF;
---color-foreground: #000000;
---color-accent: #ABABAB;
+        --color-background: #FFFFFF;
+        --color-foreground: #000000;
+        --color-accent: #ABABAB;
+        --box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
     }
+
+    * {
+        box-sizing: border-box;
+        xborder: 1px solid black
```

Comment:
here is an obsolete border property

Reply:
Ulli, thanks for reviewing my code! :)
I'm using this border property on and off to get a better overview of the rendered html elements.
Therefore I would like to keep this property.